II.pass-tests/07-second-largest/second-largest.test.js

```diff
@@ -5,5 +5,5 @@ test('Second largest', function () {
 
     var output = secondLargest(numbers);
 
-    expect(output).toEqual(2);
+    expect(output).toEqual(23);
```

Comment:
Sorry! Another mistake 😞 I'll make sure it's fixed for next time

Reply:
No worries!